### ISSUE TITLE
Нарисовать скелетонов

### ISSUE THREAD

**Issue by hippogamesunity** [hippogamesunity/SpellMasters#27]
За основу нужно взять Body человека и сохранить разметку. Анимация будет такая же, как у людей (но возможно и сделаем отдельно Stand и Alert). По сути это будет простой скин Body для редактора. Сложности могут быть со стыками, хотя у скелетов нет бесшовных соединений. Лучше нарисовать несколько разных вариаций, без градации. Рефы из Maple прикладываю. Эквип будем рисовать отдельно. Доспехи будут прикрывать минимум поверхности скелета.

![image](https://user-images.githubusercontent.com/7318172/27880607-3e6d4cc6-61ce-11e7-9656-1c0cda4ec797.png)
![image](https://user-images.githubusercontent.com/7318172/27880616-48127a8a-61ce-11e7-93b9-f3a926658bb3.png)
![image](https://user-images.githubusercontent.com/7318172/27880641-59d3a97e-61ce-11e7-8c04-2c994eef8b38.png)
![image](https://user-images.githubusercontent.com/7318172/27880657-6164f184-61ce-11e7-9321-a1ca8072c9a3.png)

**Comment by Ragewortt** [comment]
![assss](https://user-images.githubusercontent.com/22120368/29243077-3dcc8a20-7fb0-11e7-97f5-aee70ce191a8.png)

**Comment by hippogamesunity** [comment]
Да, так лучше. Можно еще на кистями ног и рук поработать, в рефах было
что-то похожее?

**Comment by Ragewortt** [comment]
На рефах ноги в основном в сапогах, а на руках по три пальца.  Можно попробовать добавить пальцы.

**Comment by Ragewortt** [comment]
![new canvas](https://user-images.githubusercontent.com/22120368/29385816-8f37a936-82f2-11e7-8c5c-733fe76cf543.png)

**Comment by hippogamesunity** [comment]
Ок, я для ног предложу свои варианты как приеду. Из скелетов лучше сделать
отдельный пак, хотя это и будет тот же самый редактор. Кости надо сделать в
чб под раскраску. Палитру редактора я расширю. Можно добавить больше
вариаций для черепов и костей.
Кроме того понадобится эквип под скелетов - ржавое старое оружие, доспехи.
Важно, чтобы доспехи оставляли как можно больше открытых участков тела
(подозреваю, что большая часть наших доспехов не подойдет для скелетов).
